### PR TITLE
Func & test for is_on_bottom()

### PR DESCRIPTION
**Pull request by alena-bartosh**
Also fix bug in `is_in_board()`

### CODE REVIEW

File: src/board.py [hunk 1]
```diff
@@ -39,3 +40,10 @@ def set_cells_with_value(self, points: list[Point], value: bool) -> None:
                 raise ValueError(f"{point} out of bounds!")
 
             self.__cells[point.x][point.y] = value
+
+    def is_on_bottom(self, figure: Figure) -> bool:
```
Comment (s373r) [inline src/board.py:44]: This method returns nothing (`None`) after `for`

File: src/point.py [hunk 2]
```diff
@@ -7,7 +7,6 @@ class Point:
     y: int
 
     def is_in_board(self, height: int, weight: int) -> bool:
-        return self.x >= 0 \
-               or self.y >= 0 \
-               or self.x < height \
-               or self.y < weight
+        is_x_in_bounds = 0 <= self.x < height
+        is_y_in_bounds = 0 <= self.y < weight
+        return is_x_in_bounds and is_y_in_bounds
```
Comment (s373r) [inline src/point.py:13]: add a bland line before `return`

File: tests/test_Board.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import unittest
+
+from board import Board
+from figure_factory import FigureFactory
+from figure_kind import FigureKind
+from point import Point
+
+
+class BoardTestCase(unittest.TestCase):
+
```
Comment (s373r) [inline tests/test_Board.py]: remove this blank line

File: tests/test_Board.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    def test_when_figure_is_not_on_bottom(self):
+        line_figure = FigureFactory.create(FigureKind.LINE)
+
```
Comment (s373r) [inline tests/test_Board.py:12]: please add a graphical representation of tests e.g.

File: src/point.py (hunk 2 above)
Comment (s373r) [reply src/point.py:13]: PS. a good catch :cat:

**Review by s373r — CHANGES_REQUESTED** [review]

File: src/board.py (hunk 1 above)
Comment (alena-bartosh) [reply src/board.py:44]: Tnx! Done

File: src/point.py (hunk 2 above)
Comment (alena-bartosh) [reply src/point.py:13]: Done

File: tests/test_Board.py (hunk 3 above)
Comment (alena-bartosh) [reply tests/test_Board.py]: Ok, but I do not like this proposition

File: tests/test_Board.py (hunk 4 above)
Comment (alena-bartosh) [reply tests/test_Board.py:12]: Done

**Review by s373r — APPROVED** [review]

**Review by Serg-Bartosh — APPROVED** [review]
WOW , it was chic

**Pull request merged**